### PR TITLE
Cut out KV Store in JES abort to sidestep a race condition.  Closes #1253

### LINKED ISSUE #1253: Cromwell .20 returns 500 on abort
I ran a super small WDL, then checked its status to make sure it was running. Then, I tried to abort it -- the request returned a 500: "The server was not able to produce a timely response to your request.". However, when I check the status of the workflow, it says it has been successfully aborted.

I included the WDL i ran against https://cromwell.gotc-int.broadinstitute.org/swagger/index.html?url=/swagger/cromwell.yaml in case you'd like to try it yourself.

task echoHelloWorld {
	command {
		echo 'Hello, World!'
	}
	runtime {
		docker: "phusion/baseimage"
		disks: "local-disk 10 HDD"
		memory: "1 GB"
		preemptible: 3
	}
}

workflow printHelloAndGoodbye {
	call echoHelloWorld
}

### PR DESCRIPTION
**Pull request by mcovarr**
Failure mode 3 from #1253

### CODE REVIEW

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala [hunk 1]
```diff
@@ -144,18 +145,11 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
...
-    case KvPair(scopedKey, operationId) if scopedKey.key == JesOperationIdKey =>
-      operationId match {
-        case Some(id) =>
-          jobLogger.info(s"$tag Aborting $id")
-          Try(Run(id, initializationData.genomics).abort()) match {
-            case Success(_) => jobLogger.info(s"$tag Aborted $id")
-            case Failure(ex) => jobLogger.warn(s"$tag Failed to abort $id: ${ex.getMessage}")
-          }
-        case None => jobLogger.warn(s"$tag Failed to abort ${jobDescriptor.call.fullyQualifiedName}. Run Id metadata field was empty.")
+      runId foreach { id =>
```
Comment (cjllanwarne) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: Was it a deliberate choice to make the receive block synchronous against the remote service?

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala [hunk 2]
```diff
@@ -144,18 +145,11 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
...
+        Try(Run(id, initializationData.genomics).abort()) match {
+          case Success(_) => jobLogger.info("{} Aborted {}", tag: Any, id)
+          case Failure(ex) => jobLogger.warn("{} Failed to abort {}: {}", tag, id, ex.getMessage)
```
Comment (cjllanwarne) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: Is there an `AbortFailedResponse`?

**Comment by cjllanwarne** [comment]
Assuming this was deliberate 👍

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala (hunk 1 above)
Comment (mcovarr) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: Yes

File: supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala (hunk 2 above)
Comment (mcovarr) [inline supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala]: Yeah, but bigger fixes are needed here per #1409

**Comment by geoffjentry** [comment]
👍

**Pull request merged**